tests/test_modeling_common.py

```diff
@@ -3546,31 +3563,39 @@ def test_sdpa_can_dispatch_composite_models(self):
                     ):
                         raise ValueError("The eager model should not have SDPA attention layers")
 
-    @parameterized.expand([("float16",), ("bfloat16",), ("float32",)])
+    @parameterized.expand(TEST_EAGER_MATCHES_SDPA_INFERENCE_PARAMETERIZATION)
     @require_torch_sdpa
-    def test_eager_matches_sdpa_inference(self, torch_dtype: str):
+    def test_eager_matches_sdpa_inference(
+        self, name, torch_dtype, padding_side, use_attention_mask, output_attentions, enable_kernels
+    ):
+        # TODO: we shouldn't need to do this skip, i.e. the test would be composable from the model tester. CLIP-like
+        # models have a custom mixin, which we detect to skip this test.
+        if not any(".ModelTesterMixin" in str(base) for base in self.__class__.__bases__):
+            self.skipTest(reason="CLIP-like models have a different `test_eager_matches_sdpa_inference`")
```

Comment:
the differences here already existed -- this test was being overwritten using the same names. 

With the parameterization we get new test names, hence this extra skip

Reply:
Ah, I didn't know changing this will make the new test names being run ... I trust you on this being neeeded here.